.mailmap

```diff
@@ -0,0 +1,8 @@
+Radovan Bast <radovan.bast@gmail.com> <bast@kth.se> 
+Radovan Bast <radovan.bast@gmail.com> <bast@users.noreply.github.com>
+Radovan Bast <radovan.bast@gmail.com> <bast@users.noreply.github.com>
+Radovan Bast <radovan.bast@gmail.com> <lastname at kth.se>
+Radovan Bast <radovan.bast@gmail.com> <radovan.bast@uit.no>
+Radovan Bast <radovan.bast@gmail.com> <rbast@users.noreply.github.com>
+Roberto Di Remigio <roberto.diremigio@gmail.com> <roberto.d.remigio@uit.no>
+Eric Noulard <eric.noulard@gmail.com> <enoulard@antidot.net>
```

Comment:
Patchwork mail address is strange.
It would be better to have only my gmail address.